pymc4/flow/transformed_executor.py

```diff
@@ -55,17 +55,17 @@ def modify_distribution(self, dist, model_info, state):
         if transformed_scoped_name in state.transformed_values:
             # We do not sample in this if branch
 
-            # 0. do not allow ambiguity in state, make sure only one value is provided to compute logp
-            if (
-                transformed_scoped_name in state.transformed_values
-                and scoped_name in state.untransformed_values
-            ):
-                raise EvaluationError(
-                    "Found both transformed and untransformed variables in the state: "
-                    "'{} and '{}', but need exactly one".format(
-                        scoped_name, transformed_scoped_name
-                    )
-                )
+            # # 0. do not allow ambiguity in state, make sure only one value is provided to compute logp
```

Comment:
I think that this shouldn't be commented out. Given that this PR will make our `trace`s always have the untransformed variables, we could make `PosteriorPredictiveSamplingExecutor` inherit from `SamplingExecutor` instead of `TransformedSamplingExecutor`.